lib/std/os/linux/io_uring.zig

```diff
@@ -1109,6 +1118,17 @@ pub fn io_uring_prep_poll_remove(
     io_uring_prep_rw(.POLL_REMOVE, sqe, -1, target_user_data, 0, 0);
 }
 
+pub fn io_uring_prep_poll_update(
+    sqe: *io_uring_sqe,
+    old_user_data: u64,
+    new_user_data: u64,
+    poll_mask: u32,
+    flags: u32,
+) void {
+    io_uring_prep_rw(.POLL_REMOVE, sqe, -1, old_user_data, flags, new_user_data);
+    sqe.rw_flags = std.mem.nativeToLittle(u32, poll_mask);
```

Comment:
What do you think of wrapping this in `io_uring_prep_poll_mask()` as liburing does, so that we can reuse for `io_uring_poll_add()` above, but more importantly also add a comment on why we do this, i.e. because poll flags were once-upon-a-time defined in the flags union as 16-bit but subsequently resized to 32-bit?

References:
https://github.com/axboe/liburing/blob/master/src/include/liburing.h#L402
https://github.com/axboe/liburing/blob/master/src/include/liburing.h#L367-L373
https://www.spinics.net/lists/io-uring/msg02848.html

Reply:
Sounds good to me. Wrapped the endian conversion and added a small doc note explaining why the conversion is needed.

Reply:
Nice, looks good.